<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fCauHinhHienThi</class>
 <widget class="QDialog" name="fCauHinhHienThi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
color:black;</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>12</y>
     <width>721</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QCheckBox" name="ckbToken">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Token</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QCheckBox" name="ckbPage">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Page</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="ckbMatKhauMail">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mật khẩu mail</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QCheckBox" name="ckbProfile">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Profile</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QCheckBox" name="ckbAds">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Ads</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="ckbCookie">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cookie</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="ckbPagePro5">
      <property name="font">
       <font>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Page pro5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QCheckBox" name="ckbBackup">
      <property name="font">
       <font>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Backup</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QCheckBox" name="ckbTuongTacCuoi">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tương tác cuối</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QCheckBox" name="ckbBM">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>BM</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="ckmEmail">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="ckbNgaySinh">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ngày sinh</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="ckbMa2FA">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mã 2FA</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QCheckBox" name="ckbTinhTrang">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tình trạng</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QCheckBox" name="ckbLocation">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Location</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="ckbTen">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tên </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="ckbPhone">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QCheckBox" name="ckbUserAgent">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Useragent</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QCheckBox" name="ckbThuMuc">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Thư mục</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QCheckBox" name="ckbHomeTown">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Hometown</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="ckbTheodoi">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Theo dõi</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="ckbGioiTinh">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Giới tính</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QCheckBox" name="ckbProxy">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Proxy</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QCheckBox" name="ckbStatus282">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Status 282</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QCheckBox" name="ckbCurrentCity">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Current City</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="ckbBanBe">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Bạn bè</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="ckbMatKhau">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mật khẩu</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QCheckBox" name="ckbNgayTao">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ngày tạo </string>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QCheckBox" name="ckbGhiChu">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ghi chú</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QCheckBox" name="ckbJob">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Job</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="ckbNhom">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Nhóm</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="ckbMailKhoiPhuc">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mail khôi phục</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QCheckBox" name="ckbAvatar">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Avatar</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QCheckBox" name="ckbDating">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Dating</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QCheckBox" name="ckbFbBlock">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Fb Block</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="ckbLuongSo">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Luồng số</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnLuu">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:darkblue;
color:white;
border-radius:20px;</string>
      </property>
      <property name="text">
       <string>Lưu</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDong">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: darkred;
border-radius:20px;
color:white;</string>
      </property>
      <property name="text">
       <string>Đóng</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
